tests/utils.js

```diff
@@ -55,7 +55,7 @@ const request = (options, { debug } = {}) => {
 
   return rpn(options).catch(err => {
     err.responseBody = err.response && err.response.body;
-    console.warn(`A request error occurred ${err.options.uri}`);
+    debug && console.warn(`A request error occurred ${err.options.uri}`);
```

Comment:
Why add the `debug` check?

Reply:
It was super noisy in the logs, especially when you expect requests to fail. Like so: https://github.com/medic/cht-core/runs/2961457948?check_suite_focus=true#step:20:487